WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewModel.swift

```diff
@@ -24,36 +19,82 @@ enum CardReaderSettingsViewActiveAlert {
     case updating
 }
 
+struct CardReaderViewModel {
```

Comment:
This public struct may prove useful for other places, like alerts, so that we are only formatting display names and battery status strings in one place.